apps/backend/src/users/user.entity.ts

```diff
@@ -13,4 +16,19 @@ export class User {
 
   @Column()
   email: string;
+
+  @Column()
+  profilePicture: string;
+
+  @Column()
+  linkedin: string | null;
+
+  @Column()
+  github: string | null;
+
+  @Column()
+  team: string | null;
```

Comment:
:pick: Can we define enums for the `team` and `role` fields (similar to `Status` for the `status` field)?

Reply:
Sure, we can also define enums for these fields. To account for people with multiple roles, would we want to change the data type to Role[]?

Reply:
Yup, that sounds good!